web/src/components/modals/columns-modal.tsx

```diff
@@ -1,35 +1,37 @@
-import * as React from 'react';
 import {
   Button,
   DataList,
+  DataListCell,
+  DataListCheck,
   DataListControl,
+  DataListDragButton,
   DataListItem,
+  DataListItemCells,
   DataListItemRow,
-  DataListDragButton,
-  DataListCheck,
-  DataListCell,
   DragDrop,
   Draggable,
   Droppable,
-  DataListItemCells,
   Text,
   TextContent,
   TextVariants,
   Tooltip
 } from '@patternfly/react-core';
-import Modal from './modal';
-import { useTranslation } from 'react-i18next';
-import { Column, getDefaultColumns, getFullColumnName } from '../../utils/columns';
 import * as _ from 'lodash';
+import * as React from 'react';
```

Comment:
out of curiosity: do you use something to reorder imports, or was that manual? Just to understand if there's a rationale for this reordering, no problem per se.
BTW I guess it would be cool, and help reduce the number of conflicts, if we all used something to reorder imports

Reply:
This comes from VSCode ``Organize imports` using `Shift` + `Alt` + `O`

We can add sort imports in eslint
https://eslint.org/docs/latest/rules/sort-imports 
and maybe need to tweak auto fix
https://github.com/aladdin-add/eslint-plugin/tree/master/packages/autofix